ui/src/shared/actions/schemaThunks.ts

```diff
@@ -30,11 +29,12 @@ type Action = BucketAction | NotifyAction
 
 // TODO(ariel): make this work with the query & the time range
 export const fetchSchemaForBucket = async (
-  // @ts-ignore
   bucketName: string,
-  // @ts-ignore
   orgID: string
 ): Promise<Schema> => {
+  /* eslint-disable no-console */
+  console.log(bucketName)
```

Comment:
kill all of this with fire or write a proper logging mechanism

Reply:
The point of it was to maintain the original function definition without getting linter errors for unused vars. Weird workaround I know but I couldn't get the other one to ignore to save my life